pkg/auth/is_jwt_token_expired.go

```diff
@@ -0,0 +1,17 @@
+package auth
```

Comment:
Cannot find a better way to put this comment: the file name doesnt follow Go conventions, "_" must not be present and the filename is specific to a method for now, would it be better to rename it as jwtUtils or something? anticpating future JWT related common utils will go in here?

Reply:
Thanks for the input @Rajagopalan-Ranganathan 
I agree, jwtUtils would be a better alternative. Also, I wonder if moving this to auth/helper.go would be out of the question?

Reply:
The debate is :) is this a helper or util method, i think this falls under util category, but please let me know yours (and others thoughts) as well

Reply:
>  Cannot find a better way to put this comment: the file name doesnt follow Go conventions, "_" must not be present and the filename is specific to a method for now, would it be better to rename it as jwtUtils or something? anticpating future JWT related common utils will go in here?

Where is the ` "_" must not be present ` in a filename convention specified? I can observe underscore being used in several well-known projects like OpenShift origin and K8s. I can also see file names with compound names containing underscore. I can't also find any mention of it in https://go.dev/doc/code#Overview.

I agree however that the file `is_jwt_token_expired` seems to specific for a file. `jwtutils.go` or `jwt_utils.go` might be a better name.

Reply:
@miguelsorianod https://go.dev/doc/effective_go#mixed-caps - it talks about names in general :). File names i assume are a part of that. But, like you mentioned i see that "_" is used in lot of big projects, so it is something we can agree on KFM level i assume.

Reply:
:+1: . mixedCaps in file names nor package names is definitely not something that I've seen (or at least seen commonly)

Reply:
Thanks for chiming in too @miguelsorianod 
After thinking over it and going through both of your points, I think jwt_utils.go would be the best naming for here. This removes the focus from a single method, allows us to add other util methods in the future, and satisfies the naming convention of what is already in place. Would you both agree?